v2/account_claims.go

```diff
@@ -61,7 +60,7 @@ type JetStreamLimits struct {
 
 // IsUnlimited returns true if all limits are unlimited
 func (j *JetStreamLimits) IsUnlimited() bool {
-	return *j == JetStreamLimits{NoLimit, NoLimit, NoLimit, NoLimit, NoLimit, false}
+	return *j == JetStreamLimits{NoLimit, NoLimit, NoLimit, NoLimit, false}
```

Comment:
JetStreamLimits will be mapped to R factor in the future, right?

Reply:
For now we will be indexing with tier names "R1" and "R3". 
However, that indexing gives us the flexibility to change that going forward.